dataregistry-extension/src/csvviewer/data_grid.ts

```diff
@@ -0,0 +1,76 @@
+/**
```

Comment:
Maybe call it CSVDataGrid to match the class? What do you think?

Reply:
Can do, if think it is useful. The main idea here is that, to a module consumer, an implementation (and name) should be opaque and having to know an exported symbol identifier means having to know something about an implementation (namely, the identifiers in that file).
​
Here, my desire in the `csv_to_data_grid.ts` file is to convert from CSV to a data grid. The activation file does not need to know that the `data_grid.ts` file exports an extended class (i.e., a specialized data grid). To it, it can simply import from `data_grid.ts` a class for creating a data grid widget. So the naming of just `data_grid.ts` was to reflect that idea: from the `data_grid.ts` file, I can import a class for creating a data grid, assuming that whatever that class is can handle CSV data.
​
I updated the exported name in `data_grid.ts` to reflect this idea.